frontmatter/default_handlers.py

```diff
@@ -12,6 +12,9 @@
 :py:class:`BaseHandler <frontmatter.default_handlers.BaseHandler>` that 
 outlines the basic API and can be subclassed to deal with new formats.
 
+**Note**: The TOML handler is only available if the `toml <https://pypi.org/project/toml/>`_
+library is installed.
```

Comment:
The cool way™ to do this is to define an [extra](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies) so that people can depend on `frontmatter[toml]` and the toml or tomli library with a compatible versions is automatically installed.